Makefile

```diff
@@ -151,8 +151,22 @@ bundle: $(KUSTOMIZE) $(OPERATOR_SDK) generate
 		 	$(BUNDLE_METADATA_OPTS)
 	$(OPERATOR_SDK) bundle validate ./bundle
 
+.PHONY: e2e-bundle
+e2e-bundle: $(KUSTOMIZE) $(OPERATOR_SDK) generate
```

Comment:
I feel we should test using the same bundle that would deployed in production to avoid any surprises.